code/game/objects/structures/transit_tubes/transit_tube_pod.dm

```diff
@@ -12,7 +12,7 @@
 /obj/structure/transit_tube_pod/New(loc)
 	..()
 	air_contents.assert_gases("o2", "n2")
-	air_contents.gases["o2"][MOLES] = MOLES_O2STANDARD * 2
+	air_contents.gases["o2"][MOLES] = MOLES_O2STANDARD
```

Comment:
The main issue is that share() compares the archived gas moles, which defaults to zero if archive() is never called and so gas is constantly being moved into the pod as though it had zero moles.